config/accounts.go

```diff
@@ -1,9 +1,61 @@
 package config
 
+// RequestType : Request type enumeration
+type RequestType string
+
+// The request types
+const (
+	RequestTypeAMP   RequestType = "AMP"
+	RequestTypeApp   RequestType = "app"
+	RequestTypeVideo RequestType = "video"
+	RequestTypeWeb   RequestType = "web"
+)
+
 // Account represents a publisher account configuration
 type Account struct {
-	ID            string      `mapstructure:"id" json:"id"`
-	Disabled      bool        `mapstructure:"disabled" json:"disabled"`
-	CacheTTL      DefaultTTLs `mapstructure:"cache_ttl" json:"cache_ttl"`
-	EventsEnabled bool        `mapstructure:"events_enabled" json:"events_enabled"`
```

Comment:
`mapstructure` was intended for parsing `account_defaults` section in the yaml config with viper. Does it still work without it ?

Reply:
Given that we currently only allow setting the account default `disabled` field on the config and it's a bool and not a struct, it doesn't look like we need the `mapstructure` annotation on the account structures. I ran some manual tests to verify you only need that annotation for non-primitive types; it wasn't needed to map bool and int config values to the structs. I'm ok with either adding or removing the annotations. @SyntaxNode @hhhjort any preference?

Reply:
I think the reason `mapstructure` isn't needed on primitives is that viper will just fall back on the property name in the struct, and it does not care about capitalization. So it is looking for "Disabled", sees "disabled" in the yaml or "DISABLED" in the env variables, and calls it a match. So the mapstructure can document the "preferred" format of the name, and help protect against a newer version of viper becoming more sensitive to capitalization.

Reply:
👍 I'll re-add those annotations.